bot/cogs/fun.py

```diff
@@ -33,12 +33,80 @@ def __init__(self, bot: Bot) -> None:
         }
         self.session = aiohttp.ClientSession()
 
+    @command()
+    async def joke(self, ctx: Context) -> None:
+        """Sends a random joke."""
+        async with self.session.get("https://mrwinson.me/api/jokes/random") as resp:
+            if resp.status == 200:
+                data = await resp.json()
+                joke = data["joke"]
+                await ctx.send(
+                    embed=Embed(
+                        description=joke,
+                        color=Color.gold()
+                    )
+                )
+            else:
+                await ctx.send(f"Something went boom! :( [CODE: {resp.status}]")
+
+    @command()
+    async def duck(self, ctx: Context) -> None:
+        """Get a random picture of a duck."""
+        async with self.session.get("https://random-d.uk/api/v2/random") as resp:
+            if resp.status == 200:
+                data = await resp.json()
+                embed = Embed(
+                    title="Random Duck!",
+                    color=Color.gold()
+                )
+                embed.set_image(url=data["url"])
+                await ctx.send(
+                    embed=embed
+                )
+            else:
+                await ctx.send(f"Something went boom! :( [CODE: {resp.status}]")
+
+    @command()
+    async def koala(self, ctx: Context) -> None:
+        """Get a random picture of a koala."""
+        async with self.session.get("https://some-random-api.ml/img/koala") as resp:
+            if resp.status == 200:
+                data = await resp.json()
+                embed = Embed(
+                    title="Random Duck!",
+                    color=Color.gold()
+                )
+                embed.set_image(url=data["url"])
+                await ctx.send(
+                    embed=embed
+                )
+            else:
+                await ctx.send(f"Something went boom! :( [CODE: {resp.status}]")
+
+    @command()
+    async def panda(self, ctx: Context) -> None:
+        """Get a random picture of a panda."""
+        async with self.session.get("https://some-random-api.ml/img/panda",) as resp:
+            if resp.status == 200:
+                data = await resp.json()
+                embed = Embed(
+                    title="Random Duck!",
+                    color=Color.gold()
```

Comment:
```suggestion
                    color=Color.gold(),
```
Either don't split it, or add a `,` at the end of each line.